### PR TITLE
Fix crash while trying to wordwrap

### PR DESCRIPTION
**Pull request by ohayon**
Fix https://github.com/fastlane/fastlane/issues/9394

### CODE REVIEW

**Review by KrauseFx — APPROVED** [review]
Nice catch 👍

**Review by KrauseFx — APPROVED** [review]
Tests ❤️

File: fastlane_core/spec/wordwrap_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    it 'will return an empty array if the length is zero' do
+      result = test_string.wordwrap(0)
+      expect(result).to eq([])
+    end
+
+    it 'will return an array of strings, each being <= the length passed to wordwrap' do
+      wrap_length = 5
+      result = test_string.wordwrap(wrap_length)
+      string_lengths = result.map { |string| string.length }
+      expect(result).to all( be_a(String) )
```
Comment (taquitos) [inline fastlane_core/spec/wordwrap_spec.rb]: is not being a String an option?

Comment (ohayon) [inline fastlane_core/spec/wordwrap_spec.rb]: not really? my thought was it might be nil if the code was changed?

Comment (taquitos) [inline fastlane_core/spec/wordwrap_spec.rb]: Oh no, you can have a nil in an array 😶

**Pull request merged**